arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -1325,5 +1325,95 @@ public async Task PutEdgeDefinitionAsync_ShouldThrow_WhenEdgeCollectionNameDoesN
             Assert.Equal(HttpStatusCode.NotFound, ex.ApiError.Code);
             Assert.Equal(1930, ex.ApiError.ErrorNum); // GRAPH_EDGE_COLLECTION_NOT_USED
         }
+
+        [Fact]
+        public async Task PatchEdgeAsync_ShouldSucceed()
+        {
+            string graphName = nameof(PatchEdgeAsync_ShouldSucceed);
+            string fromClx = graphName + "_fromclx";
+            string toClx = graphName + "_toclx";
+            string edgeClx = graphName + "_edgeclx";
+
+            // Create a new graph
+
+            await _fixture.ArangoDBClient.Graph.PostGraphAsync(new PostGraphBody
+            {
+                Name = graphName,
+                EdgeDefinitions = new List<EdgeDefinition>
+                {
+                    new EdgeDefinition
+                    {
+                        From = new string[] { fromClx },
+                        To = new string[] { toClx },
+                        Collection = edgeClx
+                    }
+                }
+            });
+
+            // Create a document in the vertex collections
+
+            PostDocumentResponse<object> fromResponse = await
+                _fixture.ArangoDBClient.Document.PostDocumentAsync<object>(
+                fromClx,
+                new { myKey = "myValue" });
+
+            PostDocumentResponse<object> toResponse = await
+                _fixture.ArangoDBClient.Document.PostDocumentAsync<object>(
+                toClx,
+                new { myKey = "myValue" });
+
+            // Create the edge
+
+            var createEdgeResponse = await _client.PostEdgeAsync(
+                graphName,
+                edgeClx,
+                new
+                {
+                    _from = fromResponse._id,
+                    _to = toResponse._id,
+                    myKey = "myValue",
+                    value = 1
+                },
+                new PostGraphEdgeQuery
+                {
+                    ReturnNew = true,
+                    WaitForSync = true
+                });
+
+            var response = await _client.PatchEdgeAsync<object, PatchEdgeMockModel>(graphName, edgeClx, createEdgeResponse.Edge._key, new
+            {
+                _from = fromResponse._id,
+                _to = toResponse._id,
+                myKey = "newValue"
+            }, new PatchEdgeQuery
+            {
+                ReturnNew = true,
+                ReturnOld = true,
+                WaitForSync = true
+            });
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.Equal(createEdgeResponse.New.myKey, response.Old.myKey);
+            Assert.NotEqual(createEdgeResponse.New.myKey, response.New.myKey);
+            Assert.False(response.Error);
+            Assert.NotEqual(response.Edge._rev, createEdgeResponse.Edge._rev);
```

Comment:
Minor: `createEdgeResponse.Edge._rev` should be provided first to the method from what I read from the others.